src/apps/test/pfutest.cpp

```diff
@@ -0,0 +1,133 @@
+#include "pfu.hpp"
+#include "../test/MemFile.hpp"
+#include "../test/fileutils.hpp"
+#include "common/Configuration.hpp"
+#include "save.hpp"
+#include "string_utils.hpp"
+
+#include <catch2/catch.hpp>
+#include <fmt/format.h>
+
+extern void boot_db();
+extern void reap_old_chars();
+
+using namespace pfu;
+
+namespace {
+spdlog::logger logger = spdlog::logger("pfutest");
+}
+
+struct LoadTinyMudOnce : Catch::TestEventListenerBase {
+    using TestEventListenerBase::TestEventListenerBase;
+
+    void testRunStarting([[maybe_unused]] Catch::TestRunInfo const &testInfo) override {
+        setenv(MUD_AREA_DIR_ENV, TEST_DATA_DIR "/area", 1);
+        setenv(MUD_DATA_DIR_ENV, TEST_DATA_DIR "/data", 1);
+        setenv(MUD_HTML_DIR_ENV, TEST_DATA_DIR "/html", 1);
+        setenv(MUD_PORT_ENV, "9000", 1);
+        boot_db();
```

Comment:
This suite boots up a tiny mud, from `limbo_test.are`. It has just enough rooms, objs and mobs to make this reasonably testable while also being fast.